pygrametl/tables.py

```diff
@@ -333,6 +333,9 @@ def ensure(self, row, namemapping={}):
         else:
             if self.rowexpander:
                 row = self.rowexpander(row, namemapping)
+                if not isinstance(row, dict):
+                    raise ValueError("The rowexpander %s did not return a row"
```

Comment:
Should this rather be a TypeError? I think so based on the following from the Python doc: "Passing arguments of the wrong type (e.g. passing a [list](https://docs.python.org/3/library/stdtypes.html#list) when an [int](https://docs.python.org/3/library/functions.html#int) is expected) should result in a [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError), but passing arguments with the wrong value (e.g. a number outside expected boundaries) should result in a [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError)."

Reply:
Fixed.